### PR TITLE
Wavect/mockups: Lightmode design alignment 

### LINKED ISSUE #377: [Gateway] : Dark & Light Mode Content should be visible.
## Issue Type

```
[x] bug report
[ ] feature request
```

## Current Behavior

Currently most of the content are not visible in light mode

## Expected Behavior

All the listed pages should show the correct content on dark and light mode.

- [ ] Bridge.
- [ ] Ecosystem
- [ ] Wallet
- [ ] History
- [ ] Earn
- [ ] Stake
- [ ] Dao
- [ ] FAQ's
- [ ] AirDrop
- [ ] BobaScope
- [ ] Lock
- [ ] Vote & Dao.
- [ ] All Modals.
- [ ] Notification and Alerts.

## Steps to Reproduce

1.
2.
3.

### LINKED ISSUE #628: [Gateway]: Update logo to new design
## Issue Type

```
[ x ] bug report
[ ] feature request
```

## Current Behavior

Current logo is older one let's get the new colored logo design for dark and light mode and update on gateway.

Need to update logo
Screenshot 2023-03-06 at 6 02 57 PM

## Expected Behavior

Use the logo design below on gateway so we can have same logo across all boba websites.
Screenshot 2023-03-06 at 6 02 50 PM

## Steps to Reproduce

1.
2.
3.

### LINKED ISSUE #648: [Gateway] Design Feedback from Yoonhee
[boba_gateway_ edits.pdf](https://github.com/bobanetwork/boba/files/10928802/boba_gateway_.edits.pdf)

### PR DESCRIPTION
**Pull request by this-dave**
## Overview

This PR is about the lightmode design alignments to fit the current figma design.
There are also a few fixes that affect light- and dark mode

## Changes
- aligned the Header to the lightmode design (own logo for lightmode, own lightmode boba icon for the menu items, Adjustments to the FeeSwitcher)
- aligned the Footer to the lightmode design (The footer is the same as darkmode, but with cyan highlighting colors)
- changes to the index/theme file to group some colors into one variable (in the theme) to make it easier for changes/alignments, as well as some additions to the right lightmode colors

#### Bridge:
- reworked some colors and implementations to fit with the figma design, mostly border colors, background colors, text colors
- fixed some designs both in darkmode/lightmode where the colors didn't match the figma design or overall design (Third Party bridges for example)

#### Ecosystem:
- adjusted some background colors and highlight colors when hovering on lightmode

#### Wallet:
- adjusted coloring, backgrounds and button design on lightmode

#### History:
- fixed the calendar on lightmode
- color/background adjustments for lightmode

#### Stake:
- "5% APY ..." is now cyan on lightmode
- color/background adjustments for lightmode
- adjusted the "Boba Balance/stake" box to the new design

#### DAO:
- color/background adjustments for lightmode
- reworked the react-select (see fixes)

### Overall fixes:
- changed the logo on darkmode to the correct logo
- reworked the "react-select" on the DAO page to a MuiSelect for the same design and to fix the design bugs
@sk-enya please take a closer look here to make sure everything works like before (Kevin and I have already tested it, so there should nothing be broken)
- reworked some .scss implementations to styled-components (more will follow in future fixes)

## Testing
- Local tested

### CODE REVIEW

File: packages/boba/gateway/src/components/select/Select.js [hunk 1]
```diff
@@ -124,53 +119,17 @@ function Select({
...
-            ...base,
-            background: theme.palette.background.secondary,
-            color: theme.palette.mode === 'light' ? theme.palette.background.default : '#fff',
-            marginRight: '5px',
-            paddingRight: '5px',
-          }),
-          valueContainer: (base) => ({
-            ...base,
-            background: theme.palette.background.default
+      <Select
```
Comment (wsdt) [inline packages/boba/gateway/src/components/select/Select.js]: @sk-enya Do you see a problem with this? We switched it with the MuiSelect to have a global styling + it has much better UX and looks better too.

**Comment by wsdt** [comment]
resolves #377 
resolves #628

**Review by wsdt — APPROVED** [review]

**Comment by sk-enya** [comment]
@this-dave can you please resolve the conflict?

File: packages/boba/gateway/src/components/select/Select.js (hunk 1 above)
Comment (sk-enya) [reply packages/boba/gateway/src/components/select/Select.js]: As far as it's working on all the places on gateway it's fine, will take look on it once again by running the code locally!

Comment (wsdt) [reply packages/boba/gateway/src/components/select/Select.js]: yeah we couldn't see an issue with it, but just wanted to be sure by asking!

Comment (sk-enya) [reply packages/boba/gateway/src/components/select/Select.js]: Need to revert this as it's breaking veDao feature which is disable as of now.

**Review by sk-enya — APPROVED** [review]
LGTM

**Comment by sk-enya** [comment]
refer to 
Screenshot 2023-03-17 at 5 24 07 AM
some snaps for light mode screens
Screenshot 2023-03-17 at 5 23 39 AM
Screenshot 2023-03-17 at 5 23 51 AM
Screenshot 2023-03-17 at 5 23 59 AM

**Review by boyuan-chen — APPROVED** [review]

**Pull request merged**